packages/material-ui/src/styles/createStyles.js

```diff
@@ -1,16 +1,3 @@
-import { createStyles as createStylesOriginal } from '@material-ui/styles';
-
-// let warnOnce = false;
-
-// To remove in v5
```

Comment:
Let's move this to v6

Reply:
Should we add a deprecation notice? It was mentioned that it's only needed for notistack. With #1824 and #18098 we will roll out our own solution to the problem, so this demo https://material-ui.com/components/snackbars/#notistack is on its way out. And notistack can clone this helper.